nixos/tests/taskserver.nix

```diff
@@ -81,7 +81,7 @@ in {
     };
 
     client1 = { pkgs, ... }: {
-      environment.systemPackages = [ pkgs.taskwarrior pkgs.gnutls ];
+      environment.systemPackages = [ pkgs.taskwarrior2 pkgs.gnutls ];
```

Comment:
Perhaps this `taskserver.nix` should be renamed to `taskserver2.nix` to highlight its purpose of testing version 2 of TaskWarrior?